protobuf: 29.3 -> 30.1

GaetanLepage opened the pull request:
## Things done

- **protobuf_30: init at 30.0**
- **protobuf: 29.3 -> 30.0**
- **protobuf: 30.0 -> 30.1**
- **protobuf: 30.2 -> 30.2**

Diff: https://github.com/protocolbuffers/protobuf/compare/v29.3...v30.1
Changelogs:
- https://github.com/protocolbuffers/protobuf/releases/tag/v30.0
- https://github.com/protocolbuffers/protobuf/releases/tag/v30.1
- https://github.com/protocolbuffers/protobuf/releases/tag/v30.2

---

- Built on platform(s)
  - [x] x86_64-linux
  - [x] aarch64-linux
  - [x] x86_64-darwin
  - [x] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [25.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) (or backporting [24.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) and [25.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#reviewing-contributions
-->

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

GaetanLepage committed:
protobuf_30: init at 30.0

GaetanLepage committed:
protobuf: 29.3 -> 30.0

Diff: https://github.com/protocolbuffers/protobuf/compare/v29.3...v30.0

Changelog:
https://github.com/protocolbuffers/protobuf/releases/tag/v30.0

SuperSandro2000 reviewed (commented):
Can we easily drop an old version?

GaetanLepage committed:
protobuf: 30.0 -> 30.1

Diff: https://github.com/protocolbuffers/protobuf/compare/v30.0...v30.1

Changelog: https://github.com/protocolbuffers/protobuf/releases/v30.1

GaetanLepage committed:
protobuf: 30.1 -> 30.2

Diff: https://github.com/protocolbuffers/protobuf/compare/v30.1...v30.2

Changelog: https://github.com/protocolbuffers/protobuf/releases/v30.2

SuperSandro2000 reviewed (approved):
LGTM
